.buildkite/scripts/post_github_comment.sh

```
@@ -0,0 +1,177 @@
+
+  echo "${header_payload}.${signature}"
+}
+
+# Function to get installation access token
+get_installation_token() {
+  local jwt_token="$1"
+  local installation_id="$2"
+
+  logger "Getting installation access token"
+
+  local response=$(curl -s \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)